bedrock/settings/base.py

```diff
@@ -517,7 +516,6 @@ def set_whitenoise_headers(headers, path, url):
     '/privacy/',
     '/foundation/',
     '/teach/',
-    '/lightbeam/',
```

Comment:
@pmac not sure if this is important or not, but I'm removing this from the `VaryNoCacheMiddleware` list for CloudFlare.

Reply:
If those URLs are gone then this is fine 👍